geode-core/src/distributedTest/java/org/apache/geode/internal/cache/ClearTXLockingDUnitTest.java

```diff
@@ -49,13 +49,14 @@
  * <p>
  * GEODE-1740: It was observed that operations performed within a transaction were not holding
  * region modification locks for the duration of commit processing. This lock is used to ensure
- * region consistency during CLEAR processing. By not holding the lock for the duration of commit
+ * region consistency during CLEAR_REGION processing. By not holding the lock for the duration of
```

Comment:
Here. Also consider fixing the formatting on the comment, since spotless has kind of messed it up following the accidental rename.

Reply:
Done.